tests/foreman/cli/test_classparameters.py

```diff
@@ -123,27 +132,50 @@ def setUpClass(cls):
         """Import some parametrized puppet classes. This is required to make
         sure that we have smart class variable available.
         Read all available smart class parameters for imported puppet class to
-        be able to work with unique entity for each specific test. Raise an
-        exception and skip all tests in case not enough parameters returned.
+        be able to work with unique entity for each specific test.
         """
         super(SmartClassParametersTestCase, cls).setUpClass()
-        cls.host_name = settings.server.hostname
-        ssh.command('puppet module install --force puppetlabs/ntp')
-        cls.env = Environment.info({u'name': 'production'})
-        Proxy.importclasses({
+        cls.puppet_module = "robottelo/cli_test_classparameters"
```

Comment:
You never use this variable directly, each call splits the value and uses only part of it. Maybe it's better to define separate variables?